<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="mainContainer">
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
                <item>
                    <widget class="QGroupBox" name="generalConfGroupBox">
                        <property name="title">
                            <string>General configuration</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="defaultChannelLabel">
                                    <property name="text">
                                        <string>Default channel</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="3">
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="lowPowerLabel">
                                    <property name="text">
                                        <string>Low Power</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="totLabel">
                                    <property name="text">
                                        <string>TOT</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QSlider" name="totSlider">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="maximum">
                                        <number>255</number>
                                    </property>
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="tickPosition">
                                        <enum>QSlider::TicksBelow</enum>
                                    </property>
                                    <property name="tickInterval">
                                        <number>16</number>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="2">
                                <widget class="QLineEdit" name="totText">
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1" colspan="2">
                                <widget class="QComboBox" name="lowPowerComboBox"/>
                            </item>
                            <item row="0" column="1" colspan="2">
                                <widget class="QComboBox" name="defaultChannelComboBox">
                                    <property name="sizeAdjustPolicy">
                                        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="channelsGroupBox">
                        <property name="title">
                            <string>Channels configuration</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_2">
                            <item row="0" column="0">
                                <widget class="QTableView" name="tableView"/>
                            </item>
                        </layout>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>640</width>
                    <height>19</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>&amp;File</string>
                </property>
                <addaction name="actionFileNew"/>
                <addaction name="actionFileOpen"/>
                <addaction name="separator"/>
                <addaction name="actionFileSave"/>
                <addaction name="actionFileSaveas"/>
                <addaction name="separator"/>
                <addaction name="actionFileClose"/>
                <addaction name="separator"/>
                <addaction name="actionFileQuit"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>&amp;Help</string>
                </property>
                <addaction name="actionHelpAbout"/>
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>&amp;Edit</string>
                </property>
                <addaction name="actionEditCut"/>
                <addaction name="actionEditCopy"/>
                <addaction name="actionEditPaste"/>
                <addaction name="separator"/>
                <addaction name="actionEditConfiguration"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuEdit"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <widget class="QToolBar" name="toolBar">
            <property name="windowTitle">
                <string>toolBar</string>
            </property>
            <property name="movable">
                <bool>false</bool>
            </property>
            <property name="allowedAreas">
                <set>Qt::TopToolBarArea</set>
            </property>
            <property name="iconSize">
                <size>
                    <width>24</width>
                    <height>24</height>
                </size>
            </property>
            <property name="floatable">
                <bool>false</bool>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionFileNew"/>
            <addaction name="actionFileOpen"/>
            <addaction name="actionFileSave"/>
            <addaction name="actionFileSaveas"/>
            <addaction name="actionFileClose"/>
            <addaction name="separator"/>
            <addaction name="actionEditConfiguration"/>
            <addaction name="separator"/>
            <addaction name="actionEditCut"/>
            <addaction name="actionEditCopy"/>
            <addaction name="actionEditPaste"/>
        </widget>
        <action name="actionFileNew">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/document/new</normaloff>:/action/document/new
                </iconset>
            </property>
            <property name="text">
                <string>&amp;New</string>
            </property>
        </action>
        <action name="actionFileOpen">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/document/open</normaloff>:/action/document/open
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Open</string>
            </property>
        </action>
        <action name="actionFileSave">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/document/save</normaloff>:/action/document/save
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Save</string>
            </property>
        </action>
        <action name="actionFileSaveas">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/document/saveas</normaloff>:/action/document/saveas
                </iconset>
            </property>
            <property name="text">
                <string>Save &amp;as</string>
            </property>
        </action>
        <action name="actionFileClose">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/document/close</normaloff>:/action/document/close
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Close</string>
            </property>
        </action>
        <action name="actionFileQuit">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/application/exit</normaloff>:/action/application/exit
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Quit</string>
            </property>
        </action>
        <action name="actionHelpAbout">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/application/about</normaloff>:/action/application/about
                </iconset>
            </property>
            <property name="text">
                <string>&amp;About</string>
            </property>
            <property name="shortcut">
                <string>F1</string>
            </property>
        </action>
        <action name="actionEditCut">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/edit/cut</normaloff>:/action/edit/cut
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Cut</string>
            </property>
        </action>
        <action name="actionEditCopy">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/edit/copy</normaloff>:/action/edit/copy
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Copy</string>
            </property>
        </action>
        <action name="actionEditPaste">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/edit/paste</normaloff>:/action/edit/paste
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Paste</string>
            </property>
        </action>
        <action name="actionEditConfiguration">
            <property name="icon">
                <iconset resource="../resources.qrc">
                    <normaloff>:/action/application/configure</normaloff>:/action/application/configure
                </iconset>
            </property>
            <property name="text">
                <string>&amp;Configuration</string>
            </property>
            <property name="shortcut">
                <string>F6</string>
            </property>
        </action>
    </widget>
    <resources>
        <include location="../resources.qrc"/>
    </resources>
    <connections/>
</ui>
